test/JDBC/input/restricting_function_overloading-vu-verify.mix

```diff
@@ -9,9 +9,6 @@ GO
 CREATE PROCEDURE OverloadedFunction AS BEGIN SELECT 1 END
```

Comment:
Let's add the test in latest/upgrade schedule if not already present.

Reply:
This test file was already added by Shalini before, not sure about adding this in latest upgrade schedule file, if it was intentionally done or missed this.